core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2929,6 +2929,13 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_FILE_SYSTEM_FINAL_TIME_UPDATE_ENABLED =
+      new Builder(Name.MASTER_FILE_SYSTEM_FINAL_TIME_UPDATE_ENABLED)
+          .setDefaultValue(true)
+          .setDescription("If false, complete file operation will not update mtime and ctime.")
```

Comment:
If true, the complete file operation will update the last modification time and last access time. When using Alluxio POSIX API, recommend setting this value to false to follow POSIX standard.